service/src/main/scala/org/apache/celeborn/server/common/http/HttpUtils.scala

```diff
@@ -50,7 +51,7 @@ object HttpUtils {
         url.getQuery
           .split("&")
           .map(_.split("="))
-          .map(arr => arr(0).toUpperCase(Locale.ROOT) -> arr(1).toUpperCase(Locale.ROOT)).toMap
+          .map(arr => arr(0).toLowerCase(Locale.ROOT) -> arr(1).toLowerCase(Locale.ROOT)).toMap
```

Comment:
Is there any particular reason to change all parameter names to lowercase instead of uppercase?

Reply:
@FMX, the parameter id is consist of `host:rpcPort:pushPort:fetchPort:replicatePort`, therefore the host should be lowercase instead of uppercase, because lowercase is better than uppercase that the value of `id` doesn't need to covert to lowercase.